### PR TITLE
Carum submission

### PR DESCRIPTION
**Pull request by stephan-nylinder**

### CODE REVIEW

File: ENA/SOP/data/.DS_Store
Comment (YvonneKallberg) [inline ENA/SOP/data/.DS_Store:1]: Delete this file?

File: ENA/SOP/.DS_Store
Comment (YvonneKallberg) [inline ENA/SOP/.DS_Store:1]: Delete this file?

Comment (stephan-nylinder) [reply ENA/SOP/.DS_Store:1]: Yup. No idea why it joined the commit. Mac specific index file.

**Review by YvonneKallberg — COMMENTED** [review]
I miss the submission of samples part. There are sections on data transfer and experiment submission, but nothing about submitting the samples. Could at least be mentioned in the process overview that the were submitted via browser. Also, it would be nice to know the steps taken to correct the samples. Did you manually download the samples, what was the command when using the script, what was input / output, etc? This batch updating of samples is something new, and worth going into details so that we know what to do next time. I've had issues previously, but thankfully fewer samples so manual edits/corrections were doable.

**Pull request merged**